arangod/VocBase/Methods/UpgradeTasks.cpp

```diff
@@ -359,27 +342,8 @@ Result createSystemStatisticsCollections(
 Result createSystemPregelCollection(TRI_vocbase_t& vocbase) {
   auto const& cname = StaticStrings::PregelCollection;
   std::shared_ptr<LogicalCollection> col;
-  auto res = methods::Collections::lookup(vocbase, cname, col);
-
-  if (res.is(TRI_ERROR_ARANGO_DATA_SOURCE_NOT_FOUND)) {
-    // if not found, create it
-
-    VPackBuilder options;
-    methods::Collections::createSystemCollectionProperties(cname, options,
-                                                           vocbase);
-    CollectionCreationInfo info{cname, TRI_COL_TYPE_DOCUMENT, options.slice()};
-    std::vector<std::shared_ptr<LogicalCollection>> cols;
-
-    OperationOptions operationOptions(ExecContext::current());
-    return methods::Collections::create(
-        vocbase, operationOptions, {info}, true, true, true,
-        nullptr /*nullptr on purpose, distributeShardsLike is defined by
-                   createSystemCollectionProperties */
-        ,
-        cols, true /* allow system collection creation */);
-  }
-
-  return {TRI_ERROR_NO_ERROR};
+  OperationOptions operationOptions(ExecContext::current());
```

Comment:
Does this somehow change how permissions are handled for creating the collections?

Reply:
Yes change unintentional.
Fixed!